src/vs/base/parts/quickopen/browser/quickOpenWidget.ts

```diff
@@ -411,7 +411,15 @@ export class QuickOpenWidget implements IModelProvider {
 		// Trigger open of element on selection
 		if (this.isVisible()) {
 			const context: IEntryRunContext = { event, keymods: this.extractKeyMods(event), quickNavigateConfiguration: this.quickNavigateConfiguration };
-			hide = this.model.runner.run(value, Mode.OPEN, context);
+			let mode = Mode.OPEN;
+
+			if (context.event instanceof KeyboardEvent) {
```

Comment:
@wprater can we just send over a flag to indicate this mode from where we call this method with the arrow key right pressed? I think we already create the standard keyboard event there and do not need to create it again here.